### PR TITLE
add ad component

### PR DESCRIPTION
**Pull request by danigrandeletgo**
Amp-ad component created

### CODE REVIEW

File: src/components/Ad.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "data-slot": string.isRequired,
+    children: oneOfType([arrayOf(node), node])
+  };
+
+  static defaultProps = {
+    children: null
+  };
+
+  get properties() {
+    return pick(this.props, ...[...Object.keys(commonPropTypes)]);
```
Comment (fedegheVerkstedt) [inline src/components/Ad.js]: am I wrong or here does not makes much sense to descructure, restructure and descructure
looks like we can write it like
```
return pick(this.props, ...Object.keys(commonPropTypes));
```

File: src/components/Ad.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    const { children } = this.props;
+    return [
+      <Helmet key="helmet">
+        <script
+          async=""
+          custom-element="amp-ad"
+          src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"
+        />
+      </Helmet>,
+      <amp-ad {...{ ...this.properties }}>{children}</amp-ad>
```
Comment (fedegheVerkstedt) [inline src/components/Ad.js]: same there?
```
<amp-ad { this.properties }>
```

File: src/components/Ad.js (hunk 1 above)
Comment (danigrandeletgo) [reply src/components/Ad.js]: this is a copy of a method used in amp-sidebar. I assumed it was ok for you if it was in the code

File: src/components/Ad.js (hunk 2 above)
Comment (danigrandeletgo) [reply src/components/Ad.js]: same here with sidebar component

File: src/components/Ad.js (hunk 1 above)
Comment (fedegheVerkstedt) [reply src/components/Ad.js]: not exactly ...in the sidebar 'id' and 'side' are added ... even though ugly.. still makes some sense :D

File: src/components/Ad.js (hunk 2 above)
Comment (fedegheVerkstedt) [reply src/components/Ad.js]: yeah ...You are right there... luckily code can also get better 👍

**Comment by fedegheVerkstedt** [comment]
It would be great as well if as a rule of thumb we could keep the README up to date at each step ...instead of digging the heap every x months 👍

File: src/components/Ad.js (hunk 1 above)
Comment (danigrandeletgo) [reply src/components/Ad.js]: done

**Comment by danigrandeletgo** [comment]
> It would be great as well if as a rule of thumb we could keep the README up to date at each step ...instead of digging the heap every x months

done

**Comment by danigrandeletgo** [comment]
> > It would be great as well if as a rule of thumb we could keep the README up to date at each step ...instead of digging the heap every x months
> 
> done

@fedegheVerkstedt could you aprove the PR please? I need to have the ad component

File: src/components/Ad.js (hunk 1 above)
Comment (fedegheVerkstedt) [reply src/components/Ad.js]: 👍

File: src/components/Ad.js (hunk 2 above)
Comment (fedegheVerkstedt) [reply src/components/Ad.js]: I'll take care of it

**Pull request merged**